test/parallel/test-tls-connect-given-socket.js

```diff
@@ -72,14 +66,14 @@ const server = tls.createServer(options, (socket) => {
 
   // Immediate death socket
   const immediateDeath = net.connect(port);
-  establish(immediateDeath).destroy();
+  establish(immediateDeath, 0).destroy();
```

Comment:
I don't think it makes sense to pass 0 to `common.mustCall()` as the `expected` argument. I would expect any `common.mustCall()` to be passed a value of 1 (or `undefined`) or higher. If the intention is that it should never be called, `common.mustNotCall()` should be used instead.